internships.html

```diff
@@ -79,21 +79,32 @@ <h2 class="heading-title">Find the right team for you</h2>
               <div class="img-outline-offset-left">
                 <img class="lazyload" data-src="/images/intern-video.jpg" alt="Video Internship"/>
                 <a class="lgx-btn lgx-btn-grey" href="#" data-toggle="modal" data-target="#lgx-modal-video">Video</a>
-              </div>
+             </div>
             </div>
             <!-- Video-->
-            <div id="lgx-modal-video" class="modal fade lgx-modal">
-              <div class="modal-dialog">
-                <div class="modal-content">
-                  <div class="modal-header">
-                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">x</button>
-                  </div>
-                  <div class="modal-body">
-                    <iframe id="modalvideo" src="https://www.youtube.com/embed/Xhs6weqDvfg" allowfullscreen></iframe>
-                  </div>
-                </div>
-              </div>
-            </div> <!-- //.Video-->
+            <body>
+
+<div class="container">
+  <!-- Modal -->
+  <div class="modal fade" id="lgx-modal-video" role="dialog">
+    <div class="modal-dialog">
+    
+      <!-- Modal content-->
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+        </div>
+        <div class="modal-body">
+         <iframe width="575" height="450"  src="https://www.youtube.com/embed/Xhs6weqDvfg" allowfullscreen></iframe>
```

Comment:
@sgha229 Can we make this `width` responsive? Probably needs to be done in a `.scss` file. I would suggest something around 70-80% width.

Reply:
In addition, `height` can be responsive as well

Reply:
Sounds good, I'll get to fixing them up.